parm/config/config.fv3.nco.static

```diff
@@ -28,6 +28,12 @@ elif [[ "$machine" = "JET" ]]; then
    export npe_node_max=24
 elif [[ "$machine" = "HERA" ]]; then
    export npe_node_max=40
+elif [[ "$machine" = "S4" ]]; then
+   if [[ "$PARTITION_BATCH" = "s4" ]]; then
+      export npe_node_max=32
+   elif [[ "$PARTITION_BATCH" = "ivy" ]]; then
+      export npe_node_max=20
+   fi
```

Comment:
I realize this is for consistency, but, `config.fv3.nco.static` is for a specific application and machine